src/runner/browser-env/vite/browser-modules/mock/@wdio-logger.ts

```diff
@@ -0,0 +1,10 @@
+export default function getLogger(): typeof console {
```

Comment:
I found that after using `addCommand` for custom commands wdio started adding its custom logs. To disable them, I implemented this stub